test/tests/parallel/set3/mta/smtp_delivery.bats

```diff
@@ -228,7 +228,7 @@ function _successful() {
 
 @test "redirects mail to external aliases" {
   _service_log_should_contain_string 'mail' 'Passed CLEAN {RelayedInbound}'
-  run bash -c "grep '<user@external.tld> -> <external1@otherdomain.tld>' <<< '${output}'"
+  assert_output --partial '<user@external.tld> -> <external1@otherdomain.tld>'
   _should_output_number_of_lines 2
```

Comment:
@polarathene this changes causes

```txt
not ok 49 [SMTP] (delivery) redirects mail to external aliases in 131ms
# (from function `assert_equal' in file test/test_helper/bats-assert/src/assert_equal.bash, line 40,
#  from function `_should_output_number_of_lines' in file test/helper/log_and_filtering.bash, line 16,
#  in test file test/tests/parallel/set3/mta/smtp_delivery.bats, line 232)
#   `_should_output_number_of_lines 2' failed
#
# -- values do not equal --
# expected : 2
# actual   : 13
# --
#
```

This is simply because `_should_output_number_of_lines 2` works on `_service_log_should_contain_string 'mail' 'Passed CLEAN {RelayedInbound}'` in your example and not on `run bash -c "grep '<user@external.tld> -> <external1@otherdomain.tld>' <<< '${output}'"` (which is the case in my example).

This may be confusing at first sight.

Reply:
I will revert this change.

Reply:
Done in 5622761c41abbf3f39bc9522d6cdfb6902ee0b88

Reply:
Resolved with https://github.com/docker-mailserver/docker-mailserver/pull/3786#discussion_r1460693476